src/navsat_transform.cpp

```diff
@@ -132,12 +130,12 @@ void NavSatTransform::run()
   }
 
   auto gps_odom_pub =
-    node_->create_publisher<nav_msgs::msg::Odometry>("odometry/gps");
+    node_->create_publisher<nav_msgs::msg::Odometry>("odometry/gps", 10);
```

Comment:
This callback depth should probably be a parameter if you're going to add it? Same with all similar below.

Reply:
Sounds good, I'll fix it.